velox/common/memory/AllocationPool.h

```diff
@@ -35,7 +35,9 @@ class AllocationPool {
 
   void clear();
 
-  char* allocateFixed(uint64_t bytes);
+  // Allocate a buffer from this pool, optionally aligned.  The alignment can
+  // only be power of 2.
+  char* allocateFixed(uint64_t bytes, int32_t alignment = 1);
```

Comment:
Would it be possible to extract this change into a separate PR to keeps PRs small and easy to review?

Reply:
The change here is just for make this PR to work.  It is well contained in its own directory (`velox/common/memory`), not mixed with other changes, so you can review it separately even in the same PR.